src/Components/Testimonials/TestimonialsForm.js

```diff
@@ -1,48 +1,132 @@
+/* eslint-disable prettier/prettier */
 import React, { useState } from "react";
 import "../FormStyles.css";
+import "bootstrap/dist/css/bootstrap.css";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import { CKEditor } from "@ckeditor/ckeditor5-react";
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+import { v4 as uuid } from "uuid";
+
+import axios from "../../api/testimonialapi";
+
+import PreviewImage from "./PreviewImage";
+import { testimonialSchema } from "./formValidation";
 
 const TestimonialForm = () => {
-  const [initialValues, setInitialValues] = useState({
-    name: "",
-    description: "",
-  });
-
-  const handleChange = (e) => {
-    if (e.target.name === "name") {
-      setInitialValues({ ...initialValues, name: e.target.value });
-    }
-    if (e.target.name === "description") {
-      setInitialValues({ ...initialValues, description: e.target.value });
-    }
-  };
-
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    console.log(initialValues);
-  };
+  const [formSend, setFormSend] = useState(false);
 
   return (
-    <form className="form-container" onSubmit={handleSubmit}>
-      <input
-        className="input-field"
-        name="name"
-        placeholder="Testimonial Title"
-        type="text"
-        value={initialValues.name}
-        onChange={handleChange}
-      />
-      <input
-        className="input-field"
-        name="description"
-        placeholder="Testimonial description"
-        type="text"
-        value={initialValues.description}
-        onChange={handleChange}
-      />
-      <button className="submit-btn" type="submit">
-        Send
-      </button>
-    </form>
+    <React.Fragment>
+      <Formik
+        validateOnMount
+        initialValues={{
+          id: uuid(),
+          name: "",
+          description: "",
+          file: null,
+        }}
+        validationSchema={testimonialSchema}
+        onSubmit={async (values, { resetForm }) => {
+          resetForm();
+          // eslint-disable-next-line no-console
+          console.log(values);
+          //Eleccion de ruta para crear o editar
+          if (!values.id || values.id === undefined) {
+            try {
```

Comment:
Si tenemos dos try and catch no valdría la pena usar solo uno, para ambos casos del if, ya que son exactamente iguales?